nemo/collections/nlp/modules/common/megatron/megatron_utils.py

```diff
@@ -33,7 +33,14 @@
 ]
 
 
-MEGATRON_CACHE = os.path.join(_get_torch_home(), "megatron")
+torch_home = _get_torch_home()
+
+if not isinstance(torch_home, str):
+    logging.info("Torch home not found, caching megatron in cwd")
+    torch_home = os.getcwd()
+
+MEGATRON_CACHE = os.path.join(torch_home, "megatron")
+
```

Comment:
hmmm, is this related to this PR?

Reply:
This is to fix NLP docs which break cause of this import. @ericharper fixed it with this workaround.